src/shogun/neuralnets/NeuralNetwork.cpp

```diff
@@ -264,7 +265,11 @@ bool CNeuralNetwork::train_gradient_descent(SGMatrix<float64_t> inputs,
 	REQUIRE(gd_learning_rate>0,
 		"Gradient descent learning rate (%f) must be > 0\n", gd_learning_rate);
 	REQUIRE(gd_momentum>=0,
-		"Gradient descent momentum (%f) must be > 0\n", gd_momentum);
+    		"Gradient descent momentum (%f) must be > 0\n", gd_momentum);
```

Comment:
I am not sure why the change in this line. Anyway, maybe we can fix the > to >= as the actual require is doing :-)